Userland/Applications/PixelPaint/ImageEditor.cpp

```diff
@@ -34,7 +34,7 @@ ImageEditor::ImageEditor(NonnullRefPtr<Image> image)
     , m_gui_event_loop(Core::EventLoop::current())
 {
     set_focus_policy(GUI::FocusPolicy::StrongFocus);
-    m_undo_stack.push(make<ImageUndoCommand>(*m_image, DeprecatedString()));
+    m_undo_stack.push(make<ImageUndoCommand>(*m_image, String::from_utf8_short_string(""sv)));
```

Comment:
I think this can just be:
    m_undo_stack.push(make<ImageUndoCommand>(*m_image, String{}));
```